main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+// Add the database path
+const DB string = "./data.csv"
+
+// Create pokemon struct
+type Pokemon struct {
+	Id int `json:"id"`
+	Name string `json:"name"`
+}
+
+// Global array for pokemons
+var Pokemons []Pokemon
```

Comment:
This could be a map that will help to filter by ID
```suggestion
var Pokemons map[int]Pokemon
```